README.md

```diff
@@ -1,82 +1,282 @@
-# Graph-Service
+# Graph Service
 
-A service enabling easy interaction with DSNP private and public graphs on Frequency
+<!-- TABLE OF CONTENTS -->
 
-## Table of Contents
+# 📗 Table of Contents
 
-- [Graph-Service](#graph-service)
-  - [Table of Contents](#table-of-contents)
-  - [Prerequisites](#prerequisites)
-  - [Getting Started](#getting-started)
-  - [Running E2E tests](#running-e2e-tests)
+- [📖 About the Project](#about-project)
+- [🔍 Arch Map](#-arch-maps)
+- [🛠 Built With](#-built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [🚀 Live OpenAPI Docs](#-live-docs)
+- [💻 Getting Started](#-getting-started)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+- [🤝 Contributing](#-contributing)
+- [⭐️ Show your support](#-support)
+- [🙏 Acknowledgements](#-acknowledgements)
+- [❓FAQ](#faq)
+- [📝 License](#-license)
 
-## Prerequisites
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 `graph-service` <a name="about-project"></a>
+
+The Graph Service is a service enabling easy interaction with DSNP private and public graphs on Frequency.
+
+<!-- Arch maps -->
+
+## 🔭 Arch Maps
+
+The account-service is a NestJS application that is split into two main parts: the API and the Worker.
+
+The API is responsible for handling incoming HTTP requests and the Worker is responsible for processing jobs that require blockchain interaction.
+
+![Gateway Graph Service](./docs/graph_service_arch.drawio.png)
+
+<p align="right">(<a href="#-table-of-contents">back to top</a>)</p>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Framework</summary>
+  <ul>
+    <li><a href="https://nestjs.com/">NestJS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Language</summary>
+  <ul>
+    <li><a href="https://www.typescriptlang.org/">Typescript</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Testing Libraries</summary>
+  <ul>
+    <li><a href="https://jestjs.io/">Jest</a></li>
+    <li><a href="https://www.npmjs.com/package/supertest">Supertest</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Linting</summary>
+  <ul>
+    <li><a href="https://eslint.org/">ESLint</a></li>
+    <li><a href="https://prettier.io/">Prettier</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Data Store</summary>
+  <ul>
+    <li><a href="https://github.com/luin/ioredis">ioredis</a></li>
```

Comment:
nit: `Redis` is the data store; `ioredis` is the JS library we use to interface with it.